src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -816,7 +816,7 @@ message GetBookFromAnywhereRequest {
   // The name of the book to retrieve.
   string name = 1 [
     (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference) = "google.example.library.v1.Book"];
+    (google.api.resource_reference) = "google.example.library.v1.BookOneOf"];
```

Comment:
This is to reflect library_gapic.yaml, which has the `GetBookFromAnywhere` method use the `book_oneof` resource set instead of the single `book` resource name.